ambry-server/src/integration-test/java/com/github/ambry/server/RouterServerTestFramework.java

```diff
@@ -141,6 +142,7 @@ void checkOperationChains(List<OperationChain> opChains) throws Exception {
         partitionCount.put(partitionId, count + 1);
       }
     }
+    /*
```

Comment:
not needed?

Reply:
I don't think this part of test is necessary. This test that newly created blobs are not created mostly in one partition. It tests the randomness of partition assignment. There is always a change that all the blobs are created in one partition if we don't somehow force it out.